src/main/java/executor/command/CommandConvert.java

```diff
@@ -80,8 +90,16 @@ private String getCurrencyCovertFrom(String userInput) {
      * @return this function return the 3 character unique string representing the currency to covert to
      */
     private String getCurrencyConvertTo(String userInput) {
-        String toStr = Parser.parseForFlag("to", userInput);
-        return toStr;
+        try {
+            String toStr = Parser.parseForFlag("to", userInput);
```

Comment:
Same as before, doesn't need to be in try-catch

Reply:
Same as before , i have removed that as well ;)